subprojects/js/emscripten-docker.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker run -it -v "$(cd ../.. && pwd)":/src trzeci/emscripten /bin/bash
```

Comment:
I want to add
--user $(id -u):$(id -g)
to make the files generated have the user id and group.